pkg/kubectl/cmd/explain.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+const (
+	explainExamples = `# Get the documentation of the resource and its fields
+$ kubectl explain pods
+
+# Get the documentation of a specific field of a resource
+$ kubectl explain pods.spec.containers
+
+# Get all available operations (GET|POST...) of a URL. {name} can be an arbitrary string.
+kubectl explain /api/v1/namespaces/{name}
+
+# Or
+kubectl explain /api/v1/namespaces/arbitrarystring
+
+# Get the documentation of the specific operation
+kubectl explain /api/v1/namespaces/{name} GET
+
+# Search for similar operations
+kubectl explain -S /api/v1/namespaces`
+	explainLong = `Documentation of resources and operations.
+
+API version in resources can be defined as a prefix: v1.RESOURCE.
```

Comment:
Does this mean you can do `kubectl explain v1.pods`? Looks like the version part isn't stripped out in the args.